arch/arc/core/cache.c

```diff
@@ -67,24 +67,7 @@ static void dcache_enable(void)
 	dcache_dc_ctrl(DC_CTRL_DC_ENABLE);
 }
 
-
-/**
- *
- * @brief Flush multiple d-cache lines to memory
- *
- * No alignment is required for either <start_addr> or <size>, but since
- * dcache_flush_mlines() iterates on the d-cache lines, a cache line
- * alignment for both is optimal.
- *
- * The d-cache line size is specified either via the CONFIG_CACHE_LINE_SIZE
- * kconfig option or it is detected at runtime.
- *
- * @param start_addr the pointer to start the multi-line flush
- * @param size the number of bytes that are to be flushed
- *
- * @return N/A
- */
-static void dcache_flush_mlines(u32_t start_addr, u32_t size)
```

Comment:
FWIW: the implementation of the new arch_cache_flush() and the old dcache_flush_mlines() look 100% identical to me (modulo that pointer dereference bug called out elsewhere).   It might be simpler from a review standpoint to limit the patch to a trivial rename (and doxygen comment removal).

Reply:
I can open a separate PR for just the rename & doxygen removal of dcache_flush_mlines() function.